src/popup/router/components/FungibleTokens/TokensList.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div>
+    <div v-if="filteredResults.length > 0">
+      <TokenDisplay
+        v-for="value in filteredResults"
+        :key="value.contract || value.id"
+        :tokenData="value"
+      />
+    </div>
+    <div v-else class="tokens-msg">{{ $t('pages.fungible-tokens.no-results') }}</div>
+  </div>
+</template>
+
+<script>
+import { mapState, mapGetters } from 'vuex';
+import TokenDisplay from './TokenDisplay';
+
+export default {
+  components: {
+    TokenDisplay,
+  },
+  props: {
+    showMyTokens: { type: Boolean },
+    searchTerm: { type: String, default: '' },
+  },
+  computed: {
+    ...mapState(['tokenBalances', 'tokenInfo', 'tokensPublicInfo']),
+    ...mapGetters(['tokenBalance', 'balanceCurrency']),
+    aeternityTokenInfo() {
+      return {
+        ...this.tokensPublicInfo[0],
+        convertedBalance: this.tokenBalance,
+        symbol: 'AE',
+        balanceCurrency: this.balanceCurrency,
+        contract: '',
+      };
+    },
+    filteredResults() {
+      let tokensList = null;
+
+      tokensList = this.tokenInfo;
+
+      if (this.showMyTokens) {
+        tokensList = [...this.tokenBalances];
+      }
+
+      const hasDefaultToken = tokensList.findIndex(({ name }) => name === 'Aeternity');
+
+      if (hasDefaultToken === -1) {
```

Comment:
following lines don't seem straight foreward for me, if they include a workaround for something it should probably be described in a comment, otherwise the code should be self-explanatory